pages/Receive.qml

```diff
@@ -291,26 +291,37 @@ Rectangle {
             }
 
             RowLayout {
-                spacing: parent.spacing
-
                 MoneroComponents.StandardButton {
-                    rightIcon: "qrc:///images/download-white.png"
+                    Layout.preferredWidth: 220
+                    small: true
+                    text: FontAwesome.save + qsTr("  Save as image") + translationManager.emptyString
```

Comment:
```suggestion
                    text: FontAwesome.save + "  %1".arg(qsTr("Save as image")) + translationManager.emptyString
```

The spaces are not interesting for translations. Better to split it up. Please also change the other.